update entry point articles for ag products (#648)
AdguardTeam/KnowledgeBase: docs/adguard-for-ios/features/features.md

## Safari protection

> Available features and settings in the *Safari protection* tab: *Content blockers*, *Filters*, *User rules*, and *Allowlist*

Reviewer: а там нет контент блокеров отдельно
```suggestion
Available features and settings in the *Safari protection* tab: *Filters*, *User rules*, and *Allowlist*
```

[Safari protection](/adguard-for-ios/features/safari-protection.md)

## DNS protection

> Settings for a secure DNS server and DNS filters to block ads, trackers and dangerous websites

Reviewer: ```suggestion
How to configure a secure DNS server and DNS filters to block ads, trackers, and dangerous websites
```

> Settings for a secure DNS server and DNS filters to block ads, trackers and dangerous websites

Reviewer: или так 
```suggestion
DNS servers and DNS filters that let you block ads, trackers, and dangerous websites
```

[DNS protection](/adguard-for-ios/features/dns-protection/)

## Advanced protection

> A module that allows AdGuard to apply advanced filtering rules and scriptlets to handle complex ads

Reviewer: хотелось бы пореже использовать слово модуль. предложила бы заменить его на feature

[Advanced protection](/adguard-for-ios/features/advanced-protection.md)

## Activity and statistics

Statistics with detailed activity reports that allow you to monitor blocked ads, trackers, and DNS queries

[Activity and statistics](/adguard-for-ios/features/activity.md)

## Assistant

> A tool that lets you quickly manage filtering in Safari right from the browser wwithout opening the AdGuard interface

Reviewer: ```suggestion
A tool that lets you quickly manage filtering in Safari right from the browser without opening the AdGuard interface
```
